src/deck/drivers/src/usddeck.c

```diff
@@ -365,17 +372,63 @@ static void usddeckEventtriggerCallback(const eventtrigger *event)
   if (logBufferReadPtr < logBufferWritePtr + dataSize)
   {
     /* write data into buffer */
-    uint16_t event_id = eventtriggerGetId(event);
+    uint16_t event_id = cfg->eventId;
     memcpy(logBufferWritePtr, &event_id, sizeof(event_id));
     logBufferWritePtr += sizeof(event_id);
     memcpy(logBufferWritePtr, &ticks, 4);
     logBufferWritePtr += 4;
-    memcpy(logBufferWritePtr, event->payload, event->payloadSize);
-    logBufferWritePtr += event->payloadSize;
+    if (payloadSize) {
```

Comment:
I do not now if there is official policy here but the brace style differs from above.

```
if (...)
{
```
vs
```
if (...) {
```

@ataffanel is there a official coding style?

Reply:
There is no project coding style, yet. However, it should be consistent within files (and ideally across the project). I updated this file to be (hopefully) consistent now.